flixel/input/mouse/FlxMouse.hx

```diff
@@ -397,8 +397,8 @@ class FlxMouse extends FlxPointer implements IFlxInputManager
 		//actually position the flixel mouse cursor graphic
 		if (visible)
 		{
-			cursorContainer.x = FlxG.game.mouseX;
-			cursorContainer.y = FlxG.game.mouseY;
+			cursorContainer.x = this.screenX;
```

Comment:
After doing some testing, it seems
```haxe
cursorContainer.x = FlxG.game.mouseX * FlxG.scaleMode.mouseMultiplier.x;
cursorContainer.y = FlxG.game.mouseY * FlxG.scaleMode.mouseMultiplier.y;
```
works, while what is currently in the pull request does not. Is there a way for me to change that easily?